circle-mlir/models/unit/Transpose_F32_R4/__init__.py

```diff
@@ -0,0 +1,21 @@
+import torch
+
+
+# Generate Transpose operator with Float32, Rank-4
+class net_Transpose(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input):
+        # NOTE two transpose will produce NCHW to NHWC single transpose
+        x = torch.transpose(input, 1, 3)
+        return torch.transpose(x, 1, 2)
```

Comment:
How about `input.permute(0,2,3,1)`?

Reply:
`input.permute()` gives same result, but I'd like to use `torch.transpose()`